api/http/handler/stacks/create_compose_stack.go

```diff
@@ -156,16 +160,29 @@ func (handler *Handler) createComposeStackFromGitRepository(w http.ResponseWrite
 
 	stackID := handler.DataStore.Stack().GetNextIdentifier()
 	stack := &portainer.Stack{
-		ID:           portainer.StackID(stackID),
-		Name:         payload.Name,
-		Type:         portainer.DockerComposeStack,
-		EndpointID:   endpoint.ID,
-		EntryPoint:   payload.ComposeFilePathInRepository,
-		Env:          payload.Env,
+		ID:              portainer.StackID(stackID),
+		Name:            payload.Name,
+		Type:            portainer.DockerComposeStack,
+		EndpointID:      endpoint.ID,
+		EntryPoint:      payload.ComposeFile,
+		AdditionalFiles: payload.AdditionalFiles,
+		AutoUpdate:      payload.AutoUpdate,
+		Env:             payload.Env,
+		GitConfig: &portainer.GitConfig{
```

Comment:
in "git stacks redeploy" we introduced `gittypes.RepoConfig` https://github.com/portainer/portainer/blob/0b93714de4864372f139aa6367ba1ff8955ac85c/api/git/types/types.go#L3

Is this the same? maybe composeFile and additionalFiles should be part of this config?

Reply:
yep, it should be the same, `GitConfig` is the name I got from Dmitry so I guess `RepoConfig` will be the final name right? I will modify this part